src/Service.Tests/Unittests/SqlQueryExecutorUnitTests.cs

```diff
@@ -318,6 +318,62 @@ Mock<MsSqlQueryExecutor> queryExecutor
             Assert.AreEqual(3, queryExecutorLogger.Invocations.Count);
         }
 
+        /// <summary>
+        /// Validates that a query successfully executes within two retries by checking that the SqlQueryExecutor logger
+        /// was invoked the expected number of times.
+        /// </summary>
+        [DataTestMethod, TestCategory(TestCategory.MSSQL)]
+        [DataRow(false, 4, DisplayName = "Data should be successfully read when amount of data being read is less than max-db-response-size-mb")]
+        [DataRow(false, 5, DisplayName = "Data should be successfully read when amount of data being read is less than max-db-response-size-mb")]
```

Comment:
data row displaynames must be unique. Otherwise they don't render as expected in test explorer in visual studio.

Reply:
fixed.